README

```diff
@@ -44,7 +44,7 @@ REQUIREMENTS:
           CONFIG_SIGNALFD
           CONFIG_TIMERFD
           CONFIG_EPOLL
-          CONFIG_NET
+          CONFIG_NET (the only other flag required is CONFIG_UNIX to include support for Unix domain sockets)
```

Comment:
maybe list CONFIG_UNIX first, CONFIG_NET second? should be self explanatory then, no?

or maybe say:

```
CONFIG_EPOLL
CONFIG_UNIX
CONFIG_NET (CONFIG_INET not necessary)
```

or so

Reply:
Yeah maybe putting only CONFIG_UNIX would suffice because it implies CONFIG_NET

Reply:
I wanted to suggest that. But I couldn't find in Kconfig where it implies that. I mean, of course I'd assume it implies it, but is that anywhere actually encoded?

Reply:
In the nconfig dialog it says CONFIG_UNIX depends on CONFIG_NET but I'm not sure it means having it would set the other.

Reply:
hmm, but I can#t find this in Kconfig. Where does "nconfig dialog" have that from?

Reply:
I think I got it. Basically in [linux/net/Kconfig](https://github.com/torvalds/linux/blob/master/net/Kconfig) there's an `if NET` that contains all other Kconfigs for the NET submenu (you can see `source "net/unix/Kconfig"` at line 64).
So if NET is not set then UNIX is not sourced. I guess nconfig/menuconfig infere the dependency from this.

I tried building a kernel with CONFIG_NET=n  and CONFIG_UNIX=y and it doesn't work so the latter requires the former.